src/main/java/com/sleekbyte/tailor/output/Printer.java

```diff
@@ -25,23 +25,21 @@
 
     private File inputFile;
     private Severity maxSeverity;
+    private Formatter formatter;
     private Map<String, ViolationMessage> msgBuffer = new HashMap<>();
     private Set<Integer> ignoredLineNumbers = new HashSet<>();
-    private ColorSettings colorSettings;
-    private int highestLineNumber = 0;
-    private int highestColumnNumber = 0;
 
     /**
      * Constructs a printer for the specified input file, maximum severity, and color setting.
      *
      * @param inputFile the source file to verify
```

Comment:
Nitpick: Phrases after the parameter names should be capitalized just like in `XcodeFormatter.java`